frontend/src/components/ProjectList/common/layout.ts

```diff
@@ -0,0 +1,8 @@
+// (thuang): Unit is %
```

Comment:
I put this file back to under `ProjectList` component, since only this component needs these constants